### PR TITLE
Add pretty labels for pass/fail

### PR DESCRIPTION
**Pull request by sofiahag**
This was suggested by Phil 8 years ago 😆 
https://github.com/NationalGenomicsInfrastructure/ngi_reports/issues/42

I'll just make a PR and we can discuss it. I'll need to change the text files to "Pass/Fail" also.

Maybe something like this?

Skärmavbild 2023-07-17 kl  13 37 15

### CODE REVIEW

**Comment by sofiahag** [comment]
giphy

File: ngi_reports/ngi_reports.py [hunk 1]
```diff
@@ -110,7 +110,10 @@ def markdown_to_html(report_type, jinja2_env=None, markdown_text=None, markdown_
                                         meta={key: ''.join(value) for (key, value) in md_template.Meta.items()})
     replace_list = {'[swedac]': swedac_text,
                     '[tick]'  : '<span class="icon_tick">&#10004;</span> ',
-                    '[cross]' : '<span class="icon_cross">&#10008;</span> '
+                    '[cross]' : '<span class="icon_cross">&#10008;</span> ',
+                    'Pass' : '<span class="pass">Pass</span>',
+                    'Fail' : '<span class="fail">Fail</span>',
+                    'NA' : '<code>NA</code>'
```
[the remark this answers is not in the document]
Comment (sofiahag) [reply ngi_reports/ngi_reports.py]: Yes!

**Review by aanil — APPROVED** [review]
👍

**Pull request merged**